packages/types/src/types.ts

```diff
@@ -401,7 +468,10 @@ export type AssistantPackageJson = PackageJson &
 /**
  * Platforms that can run Assistants.
  */
-export type Platform = 'sketch' | 'node'
+export enum Platform {
```

Comment:
Not specific to this PR but I just wondered if _platform_ is the correct terminology in computing terms. Aren't platforms more something like macOS, Linux, Windows, etc. whereas `sketch` and `node` would be engines or JavaScript environments, especially since Node is cross-platform?

Reply:
Yep perhaps you're right. V8 would be an example of a JavaScript _engine_ though, which powers both Node and Sketch's JavaScript environment.

Perhaps then `runtime`? With values of `node` or `sketch`?

Reply:
Yeah, runtime is better 👍